migrations/v3.js

```diff
@@ -0,0 +1,209 @@
+    const isValid = courseGlossaryGlobals.labelLink === 'Terms beginning with';
+    if (!isValid) throw new Error('Glossary - global attribute labelLink');
+    return true;
+  });
+
+  checkContent('Glossary - check global attribute labelNavigation', async (content) => {
+    const isValid = courseGlossaryGlobals.labelNavigation === 'Glossary navigation';
+    if (!isValid) throw new Error('Glossary - global attribute labelNavigation');
+    return true;
+  });
+
+  checkContent('Glossary - check course attribute title', async (content) => {
+    const isValid = course._glossary.title !== '';
+    if (!isValid) throw new Error('Glossary - course attribute title');
+    return true;
+  });
+
+  checkContent('Glossary - check course attribute description', async (content) => {
+    const isValid = course._glossary.description !== '';
+    if (!isValid) throw new Error('Glossary - course attribute description');
+    return true;
+  });
+
+  checkContent('Glossary - check default clearSearch', async (content) => {
+    const isValid = course._glossary.clearSearch !== '';
+    if (!isValid) throw new Error('Glossary - course attribute clearSearch');
+    return true;
+  });
+
+  checkContent('Glossary - check default searchItemsAlert', async (content) => {
+    const isValid = course._glossary.searchItemsAlert !== '{{filteredItems.length}} found.';
+    if (!isValid) throw new Error('Glossary - course attribute searchItemsAlert');
+    return true;
+  });
+
+  checkContent('Glossary - check default searchPlaceholder', async (content) => {
+    const isValid = course._glossary.searchPlaceholder !== '';
+    if (!isValid) throw new Error('Glossary - course attribute searchPlaceholder');
+    return true;
+  });
+
+  checkContent('Glossary - check default searchWithInDescriptionLabel', async (content) => {
+    const isValid = course._glossary.searchWithInDescriptionLabel !== '';
+    if (!isValid) throw new Error('Glossary - course attribute searchWithInDescriptionLabel');
+    return true;
+  });
+
+  checkContent('Glossary - check attribute _isSearchEnabled', async (content) => {
+    const isValid = course._glossary._isSearchEnabled === true;
+    if (!isValid) throw new Error('Glossary - course attribute _isSearchEnabled');
+    return true;
+  });
+
+  checkContent('Glossary - check attribute _isIndexEnabled', async (content) => {
+    const isValid = course._glossary._isIndexEnabled === false;
+    if (!isValid) throw new Error('Glossary - course attribute _isIndexEnabled');
+    return true;
+  });
+
+  checkContent('Glossary - check attribute _isGroupHeadersEnabled', async (content) => {
+    const isValid = course._glossary._isGroupHeadersEnabled === false;
+    if (!isValid) throw new Error('Glossary - course attribute _isGroupHeadersEnabled');
+    return true;
+  });
+
+  updatePlugin('Glossary - update to v3.0.0', { name: 'adapt-contrib-glossary', version: '3.0.0', framework: '">=5' });
```

Comment:
```suggestion
  updatePlugin('Glossary - update to v3.0.0', { name: 'adapt-contrib-glossary', version: '3.0.0', framework: '">=5.0.0' });
```

Reply:
Not sure if this is more appropriate or if it truly needs to match the bower.json file at the time.

Reply:
Yeah, not sure, @joe-allen-89 any thoughts? What is the convention?

Reply:
@chris-steele whilst this one won't fail through semver I'd say update to full version for consistency as I think nearly every version has been updated to full.